images/nested-libvirt/README.md

```diff
@@ -0,0 +1,50 @@
+# Nested libvirt on GCE
+
+This image enables launching a libvirt cluster nested in a GCE instance, for libvirt CI testing.
+
+This image contains [`nss_wrapper`](https://cwrap.org/nss_wrapper.html) to execute `ssh` commands as
+a mock user to interact with a GCE instance from an OpenShift container.
+
+OpenShift containers run with an arbitrary uid, but SSH requires a valid user.  `nss_wrapper`
+allows for the container's user ID to be mapped to a username inside of a container.
+
+### Example Usage
+
+You can override the container's current user ID and group ID by providing `NSS_WRAPPER_GROUP` and `NSS_WRAPPER_PASSWD` for the mock files, as well as `NSS_USERNAME`, `NSS_UID`, `NSS_GROUPNAME`, and/or `NSS_GID`. In OpenShift CI, `NSS_USERNAME` and `NSS_GROUPNAME` are set.  The random UID assigned to the container is the UID that the mock username is mapped to. 
+
+```sh
+podman run --rm -it \
+  -e NSS_WRAPPER_GROUP=/tmp/group \
+  -e NSS_WRAPPER_PASSWD=/tmp/passwd \
+  -e NSS_UID=1000 \
+  -e NSS_GID=1000 \
+  -e NSS_USERNAME=testuser \
+  -e NSS_GROUPNAME=testuser \
+  nss_wrapper_img mock-nss.sh id testuser
+
+uid=1000(testuser) gid=1000(testuser) groups=1000(testuser)
```

Comment:
This is command output, so you need either:

    ```console
    $ podman run -rm -e NSS_WRAPPER_GROUP=/tmp/group ... nss_wrapper_img mock-nss.sh id testuser
    uid=1000(testuser) gid=1000(testuser) groups=1000(testuser)
    ```

or:

    ```console
    $ podman run -rm \
    >   -e NSS_WRAPPER_GROUP=/tmp/group \
    >   ... \
    >   nss_wrapper_img mock-nss.sh id testuser
    uid=1000(testuser) gid=1000(testuser) groups=1000(testuser)
    ```

or:

    ```sh
    podman run -rm \
      -e NSS_WRAPPER_GROUP=/tmp/group \
      ... \
      nss_wrapper_img mock-nss.sh id testuser
    ```

    which produces:

    ```
    uid=1000(testuser) gid=1000(testuser) groups=1000(testuser)
    ```

More on `console` vs. `sh` in #27.